lib/subscription.js

```diff
@@ -21,12 +21,18 @@ function createConnectionHandler ({ schema, subscriber, fastify, lruGatewayResol
       return
     }
 
+    let context = {}
+    if (subscriptionContextFn) {
+      context = await subscriptionContextFn(connection, request)
```

Comment:
I would prefer if these would properties a single object instead.

Also, could you add a test where this function throws an error? Thanks